### ISSUE TITLE
No upload on file save

### ISSUE THREAD

**Issue by daizumi** [h3imdall/ftp-remote-edit#150]
I successfully connected to my server using SFTP (SHH) and could navigate my tree and download files. However, after saving files, nothing happens. The log only shows files being downloaded (Direction <--) but none being uploaded (Direction -->, I suppose, but I haven't seen that). When I close the file and open it again, changes have not been applied, of course.

**Comment by cschindl** [comment]
Hi @daizumi,

I apologize for the late answer. To help you with your problem, I need some information:

Have you checked the access rights on the server and maybe you tried to reconstruct the situation with another FTP program like Filezilla?
Have you activated the developer mode and can you send me the output from the console? (see https://github.com/h3imdall/ftp-remote-edit#package-preferences)

**Comment by daizumi** [comment]
Hi @cschindl, thanks for the reply!

Using the Developer Console (sorry, I just learned how to do so a minute ago) I realized the error is being caused by another plugin - [save-commands](https://github.com/JsonHunt/save-commands) - which throws an error on save and doesn't allow ftp-remote-edit to continue with saving. @JsonHunt, the owner of the project says the [issue](https://github.com/JsonHunt/save-commands/issues/38) has already been resolved, but somehow it still breaks my workflow, so I will report the issue there again.
